src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_unicast_transports.cs

```diff
@@ -71,7 +71,11 @@ public class SubscriberA : EndpointConfigurationBuilder
         {
             public SubscriberA()
             {
-                EndpointSetup<DefaultServer>(c => { c.RegisterPublisher(typeof(MyEvent), PublisherEndpoint); });
+                EndpointSetup<DefaultServer>((c, r) =>
+                {
+                    var routing = c.UseTransport(r.GetTransportType()).Routing();
+                    routing.RegisterPublisher(typeof(MyEvent), PublisherEndpoint);
```

Comment:
you can now use `c.MessageDrivenPubSubRouting().RegisterPublisher(typeof(MyEvent), PublisherEndpoint)` instead :)